### PR TITLE
Typo fix plus improvement of Overview

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
@nikitin-da PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ Currently in development.
 * Powerful & Simple set of Operations: `Put`, `Get`, `Delete`
 * API for Humans: Type Safety, Immutability & Thread-Safety
 * Convenient builders with compile-time guarantees for required params. Forget about 6-7 `null` in queries
-* Object mapping, if you don't want to work with `Cursor` and `ContentValue` you don't have to
-* No reflection, no annotations, `StorIO` is not ORM
```
Comment (nikitin-da) [inline README.md]: actually yes to annotations =)

**Comment by nikitin-da** [comment]
Cool! How have you caught it?

**Comment by artem-zinnatullin** [comment]
What do you mean?

**Comment by nikitin-da** [comment]
'ContentValue' instead of 'ContentValues'

**Comment by artem-zinnatullin** [comment]
Just saw it :)

**Pull request merged**